app/Http/Resources/ApplicationSettings.php

```diff
@@ -64,6 +65,7 @@ public function toArray($request)
                     'link_target' => setting('banner.link_target'),
                 ])
             ],
+            'timezones' => timezone_identifiers_list(),
```

Comment:
I'm not sure if this is the best idea. This will increase the size of the settings request from 1,8 kB before to 10,2 kB. Do you think a separate route with just the timezones would be better?